examples/runtime-sdk/minimal-runtime/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "minimal-runtime"
-version = "0.1.0"
-edition = "2018"
+version = "0.0.0"
```

Comment:
Weren't there some issues with net-runner and version being 0.0.0?

Reply:
Ah you're right! Let me change this back.